pkg/rancher-desktop/components/ContainerLogs.vue

```diff
@@ -0,0 +1,381 @@
+            selection:     'rgba(139, 233, 253, 0.3)',
+            black:         '#000000',
+            red:           '#ff5555',
+            green:         '#50fa7b',
+            yellow:        '#f1fa8c',
+            blue:          '#8be9fd',
+            magenta:       '#ff79c6',
+            cyan:          '#8be9fd',
+            white:         '#f8f8f2',
+            brightBlack:   '#6272a4',
+            brightRed:     '#ff6e6e',
+            brightGreen:   '#69ff94',
+            brightYellow:  '#ffffa5',
+            brightBlue:    '#d6acff',
+            brightMagenta: '#ff92df',
+            brightCyan:    '#a4ffff',
+            brightWhite:   '#ffffff',
+          },
+          fontSize:     14,
+          fontFamily:   '\'Courier New\', \'Monaco\', monospace',
+          cursorBlink:  false,
+          disableStdin: true,
+          convertEol:   true,
+          scrollback:   50000,
+          wordWrap:     true,
+        });
+
+        this.fitAddon = new FitAddon();
+        this.terminal.loadAddon(this.fitAddon);
+
+        this.searchAddon = new SearchAddon();
+        this.terminal.loadAddon(this.searchAddon);
+
+        this.terminal.loadAddon(new WebLinksAddon((event, uri) => {
+          event.preventDefault();
+          shell.openExternal(uri);
+        }));
+
+        this.terminal.open(this.$refs.terminalContainer);
+
+        await this.$nextTick();
+        this.fitAddon.fit();
+
+        // this.terminal.write('\x1b[?25l');
+
+        this.resizeHandler = () => {
+          this.fitAddon?.fit();
+        };
+        window.addEventListener('resize', this.resizeHandler);
+
+        if (window.ResizeObserver) {
+          this.resizeObserver = new ResizeObserver(() => {
+            this.fitAddon?.fit();
+          });
+          this.resizeObserver.observe(this.$refs.terminalContainer);
+        }
+      }
+    },
+    async startStreaming() {
+      try {
+        this.error = null;
+        this.waitingForInitialLogs = true;
+        this.hasReceivedLogs = false;
+
+        if (!this.terminal) {
+          await this.initializeTerminal();
+        }
+
+        const streamOptions = {
+          cwd:    '/',
+          stream: {
+            onOutput: (data) => {
+              if (this.terminal && (data.stdout || data.stderr)) {
+                const output = data.stdout || data.stderr;
+
+                this.hasReceivedLogs = true;
+
+                this.terminal.write(output);
+
+                if (this.waitingForInitialLogs) {
+                  if (this.revealTimeout) {
+                    clearTimeout(this.revealTimeout);
+                  }
+
+                  this.revealTimeout = setTimeout(() => {
+                    this.waitingForInitialLogs = false;
+                    this.$nextTick(() => {
+                      if (this.fitAddon) {
+                        this.fitAddon.fit();
+                      }
```

Comment:
```suggestion
                      this.fitAddon?.fit();
```
Similarly in `clearSearch`.